lib/mix/lib/mix/tasks/deps.clean.ex

```diff
@@ -53,8 +53,8 @@ defmodule Mix.Tasks.Deps.Clean do
           Mix.raise(
             "\"mix deps.clean\" expects dependencies as arguments or " <>
               "a flag indicating which dependencies to clean. " <>
-              "The --all option will clean all dependencies while " <>
-              "the --unused option cleans unused dependencies"
+              "The --all flag will clean all dependencies while " <>
+              "the --unused flag cleans unused dependencies"
```

Comment:
I would stick with `option` or `switch` here. It may be worth standardizing on one of the two.

Reply:
After reading the `OptionParser` module documentation, I think I would use “switch” for the command line (`--all`) and “option” for the keyword list. So, the `OptionParser`  transforms switches from the command line into options. Does that make sense?

Reply:
Said that, looking at the documentation of several Mix tasks, the term “option” dominates.

Reply:
the thing is that if we do not use flag, there are other instances of options with no arguments where it is referred to as "flag", and we should change it.
What this commit does is to standardize the use of "flag" so when there is an option with no argument we refer to it as "flag" for clarity, but If we decide to call it "option" when we should replace it wherever we call it "flag"

Reply:
I still think having three different names (switch, option, flag) for the same thing is confusing. Of those, I think flag is the least common, so I would completely remove it from the codebase. I definitely do not want to end-up using flag in places we didn't before. :) Thoughts?

Reply:
I have created a new commit that replaces flag. Please have a look